src/main/java/ladysnake/blast/common/world/CustomExplosion.java

```diff
@@ -300,12 +310,18 @@ public void affectWorld(boolean boolean_1) {
         }
     }
 
+    public boolean canInteract(ServerWorld world, ServerPlayerEntity player, BlockPos pos, ClaimPermission type) {
```

Comment:
This will crash with a `ClassDefNotFoundError` when Flan is not loaded. You need to at least move this method to a separate class that never gets loaded unless Flan is present.